application/src/main/java/org/togetherjava/tjbot/features/moderation/TransferQuestionCommand.java

```diff
@@ -153,21 +164,20 @@ private RestAction<User> createForumPost(ModalInteractionEvent event, User origi
         return questionsForum.createForumPost(forumTitle, forumMessage)
             .setTags(ForumTagSnowflake.fromId(defaultTag.getId()))
             .map(ForumPost::getMessage)
-            .flatMap(message -> message.reply("Original Post from " + originalUser.getAsMention()))
+            .flatMap(message -> message.reply(originalUser.getAsMention()))
             .map(sent -> originalUser);
     }
 
     private RestAction<Message> dmUser(MessageChannelUnion sourceChannel, User originalUser,
             Guild guild) {
 
         return originalUser.openPrivateChannel()
-            .flatMap(channel -> channel.sendMessage(
-                    "Hi, TJ here. You are getting this message because you tried to ask a question in wrong chat, please read rules - %s"
-                        .formatted(guild.getName())))
-            .onErrorFlatMap(error -> sourceChannel.sendMessage(
-                    "Due to failed dm interaction, you are getting this message here. Please read rules about asking questions -%s%s"
-                        .formatted(guild.getName(), originalUser.getAsMention())));
-
+            .flatMap(channel -> channel.sendMessage(String.format(
+                    "Hello 👋 You have asked a question on %s in the wrong channel %n Not a big deal, but none of the experts who could help you are reading your question there 🙁 %n Your question has been automatically transferred to %s, please continue there. %n You might also want to give #welcome a quick read, thank you \uD83D\uDC4D",
+                    guild.getName(), getHelperForum(guild.getJDA()))))
+            .onErrorFlatMap(error -> sourceChannel.sendMessage(String.format(
+                    "Hello %s 👋 You have asked a question in the wrong channel %n Not a big deal, but none of the experts who could help you are reading your question there 🙁 %n Your question has been automatically transferred to %s, please continue there. %n You might also want to give #welcome a quick read, thank you \uD83D\uDC4D",
```

Comment:
* Whats that `\uD83D\uDC4D`? Put smileys directly in the code, no unique-escapes please
* prefer text-blocks instead of all that `%n`. so `"""`
* the first sentence lacks a dot (wrong channel[.] Not a big)